src/Nethermind/Nethermind.Blockchain/ISyncConfig.cs

```diff
@@ -29,6 +29,9 @@ public interface ISyncConfig : IConfig
         [ConfigItem(Description = "If set to 'true' then the Fast Sync (eth/63) synchronization algorithm will be used.", DefaultValue = "false")]
         bool FastSync { get; set; }
         
+        [ConfigItem(Description = "Relevant only if 'FastSync' is 'true'. If set to value, then it will set a minimum height threshold limit up to witch Full Sync will stay on when chain will be behind network.", DefaultValue = "null")]
+        long? FastSyncCatchUpHeightDelta { get; set; }
```

Comment:
The description and name is unclear.
Typo in 'which'.

Reply:
The value set here defines the block number difference between the best synchronized block in this node and the best block known among the peers that causes the synchronization mode to switch back to fast sync from a full sync mode. If set to null then the node will never go back to fast sync after reqching the full sync mode.

Reply:
No, if set to null it will always switch (assuming FullSyncTreshold would be met)